commands/deploy_command.go

```diff
@@ -227,18 +231,79 @@ func (c *DeployCommand) Execute(args []string) ExecutionStatus {
 		return Failure
 	}
 
+	isUrl := strings.Contains(mtaArchive, "://")
```

Comment:
Have you thought of better way for detecting if the string is a url? Because a possible scenario is to specify the url without the protocol.